src/components/Comments/Comment.js

```diff
@@ -306,7 +376,6 @@ const styles = StyleSheet.create({
     color: colorBasedVote
   }),
   btn: {
-    // width: 30,
     height: 30,
     justifyContent: 'center',
     alignItems: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The width property has been removed from the button style. This could potentially affect the layout of your application. Make sure to test this thoroughly on different screen sizes and resolutions to ensure it doesn't cause any unexpected issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->